### PR TITLE
wine 1.8

### PR DESCRIPTION
**Pull request by uyjulian**

### CODE REVIEW

**Comment by DomT4** [comment]
```
wine:
 * stable and devel versions are identical
 * 68: Trailing whitespace was found
 * Don't negate 'build.without?': use 'build.with?'
```

File: Library/Formula/wine.rb [hunk 1]
```diff
@@ -28,35 +19,15 @@ class Wine < Formula
...
-    end
-
-    # Patch to fix texture compression issues. Still relevant on 1.7.53.
-    # https://bugs.winehq.org/show_bug.cgi?id=14939
-    patch do
-      url "https://bugs.winehq.org/attachment.cgi?id=52384"
-      sha256 "30766403f5064a115f61de8cacba1defddffe2dd898b59557956400470adc699"
-    end
-  end
+  # devel do
```
Comment (apjanke) [inline Library/Formula/wine.rb]: This `devel` block could just be removed instead of commented out.

File: Library/Formula/wine.rb [hunk 2]
```diff
@@ -119,7 +105,7 @@ def wine_wrapper; <<-EOS.undent
   end
 
   def install
-    ENV.m32 # Build 32-bit; Wine doesn't support 64-bit host builds on OS X.
+    ENV.m32 if build.without? "win64"
```
Comment (DomT4) [inline Library/Formula/wine.rb]: We should probably leave this line how it was for now, since the `with-win64` option is a no-op right now on OS X.

**Comment by alexchandel** [comment]
Rerun the checks?

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/3d4c968405d4c5ea0e03f6b7e7209243aa8d303c. Thank you for another contribution to Homebrew @uyjulian! :santa:

**Comment by rhendric** [comment]
@uyjulian, is it intentional that [line 138](https://github.com/Homebrew/homebrew/pull/47188/files#diff-5eaa5cf9d6ebc5bdcf9be58de0f9b0efR138) is still in there? It's failing the build for me (but my machine's old; could be a Snow Leopard thing). Looks a little questionable to keep something that had been guarded by a `build.stable?` check across the version jump, is all.

**Comment by uyjulian** [comment]
Nope, that isn't intentional. I'll send a fix in.

**Pull request closed without merging**